test/integration/start_page_test.rb

```diff
@@ -0,0 +1,26 @@
+require_relative "../integration_test_helper"
+require_relative "../helpers/fixture_flows_helper"
+
+class StartPageTest < ActionDispatch::IntegrationTest
```

Comment:
This integration test is not describing the behaviour of the different types of start buttons we have. Everything is hidden behind FixtureFlowsHelper.

I would expect to have something like:

```ruby
describe "start_page" do
  context "when the smart answer is `overseas-passports`" do
  end
  context "when the smart answer is `calculate-your-child-maintenance`" do
  end
  context "otherwise" do
  end
end
```